javascript/javascript2/week2/script.js

```diff
@@ -0,0 +1,98 @@
2005,"rating": 5.8,"votes": 19794,"running_times": 5820},{"title": "Last Holiday","year": 2006,"rating": 6.4,"votes": 23312,"running_times": 6720},{"title": "Last Knights","year": 2015,"rating": 6.2,"votes": 32149,"running_times": 6900},{"title": "Last Man Standing","year": 1996,"rating": 6.4,"votes": 47431,"running_times": 6060},{"title": "Last Night","year": 2010,"rating": 6.6,"votes": 42441,"running_times": 5580},{"title": "Last Shift","year": 2014,"rating": 5.8,"votes": 12443,"running_times": 5400},{"title": "Last Vegas","year": 2013,"rating": 6.6,"votes": 115090,"running_times": 6300},{"title": "Lat sau san taam","year": 1992,"rating": 7.9,"votes": 41050,"running_times": 7860},{"title": "Latter Days","year": 2003,"rating": 7.1,"votes": 15357,"running_times": 6420},{"title": "Laura","year": 1944,"rating": 8.1,"votes": 35204,"running_times": 5280},{"title": "Laurel Canyon","year": 2002,"rating": 6.4,"votes": 13693,"running_times": 6180},{"title": "Laurence Anyways","year": 2012,"rating": 7.6,"votes": 13696,"running_times": 10080},{"title": "Lava","year": 2014,"rating": 7.2,"votes": 11034,"running_times": 420},{"title": "Law Abiding Citizen","year": 2009,"rating": 7.4,"votes": 234737,"running_times": 7080},{"title": "Lawless","year": 2012,"rating": 7.3,"votes": 201216,"running_times": 6960},{"title": "Lawrence of Arabia","year": 1962,"rating": 8.3,"votes": 215696,"running_times": 13680},{"title": "Laws of Attraction","year": 2004,"rating": 5.9,"votes": 19929,"running_times": 5400},{"title": "Layer Cake","year": 2004,"rating": 7.4,"votes": 143588,"running_times": 6300},{"title": "Le cinquième élément","year": 1997,"rating": 7.7,"votes": 374767,"running_times": 7560},{"title": "Le divorce","year": 2003,"rating": 4.9,"votes": 10178,"running_times": 7020},{"title": "Le fabuleux destin d'Amélie Poulain","year": 2001,"rating": 8.3,"votes": 594927,"running_times": 7320},{"title": "Le grand bleu","year": 1988,"rating": 7.6,"votes": 41252,"running_times": 10080},{"title": "Le locataire","year": 1976,"rating": 7.8,"votes": 30919,"running_times": 7560},{"title": "Le mépris","year": 1963,"rating": 7.7,"votes": 22380,"running_times": 6180},{"title": "Le procès","year": 1962,"rating": 7.8,"votes": 15401,"running_times": 7140},{"title": "Le salaire de la peur","year": 1953,"rating": 8.2,"votes": 40345,"running_times": 9360},{"title": "Le violon rouge","year": 1998,"rating": 7.7,"votes": 28394,"running_times": 8400},{"title": "Lean on Me","year": 1989,"rating": 7.4,"votes": 15573,"running_times": 6480},{"title": "Leap Year","year": 2010,"rating": 6.5,"votes": 78668,"running_times": 6000},{"title": "Leap of Faith","year": 1992,"rating": 6.1,"votes": 10526,"running_times": 6480},{"title": "Leatherface: Texas Chainsaw Massacre III","year": 1990,"rating": 5.2,"votes": 11025,"running_times": 5100},{"title": "Leatherheads","year": 2008,"rating": 6,"votes": 28907,"running_times": 6840},{"title": "Leaves of Grass","year": 2009,"rating": 6.5,"votes": 24966,"running_tim
```

Comment:
nice work! What do you here is create a new array which has the same movie objects, however you've added an additional attribute called tag.
So when you iterate through the movies with .map it would make more sense instead of 'tag' to have:
```
movies.map((movie) => {....movie.tag = "Good"....});
```